workspace/test_configurations.go

```diff
@@ -90,6 +92,8 @@ var TestConfigurations = map[string]TestConfiguration{
 	"coffeescript": {
 		Command: "jasmine-node --coffee {{test_files}}",
 	},
+	// common-lisp: tests are loaded into a "running Lisp implementation", not the CLI directly
+	// cpp: tests are mostly run via IDE; only linux seems to support `make`. There's also a more complex test creation process (using `CMake`)
```

Comment:
Could we do cmake and then make via `&&`? See https://github.com/exercism/cpp-test-runner/blob/main/bin/run.sh#L43 We'd have to test if this works in Windows of course.

Reply:
It won't work as written. It works if I run `cmake . && make` manually, but because we're passing the command into a single `exec.Command`, it's being run as `cmake . make`, and `make` is ignored as an unknown path:

```
CMake Warning:
  Ignoring extra path from command line:

   "make"
```

To make that work, we'd need to make every command a slice of strings (most of which will only have 1 item) and then run each in order (which will work cross-platform). 

I think that's a reasonable improvement to make, but I'd recommend putting it in its own PR; this one has gotten big. I also have no idea if/how this build script works on windows- I left off the `-G "Unix Makefiles"` part of the command and I have no idea how important that is. Verifying this is probably best left to someone who knows more about those tracks.

Reply:
Actually, it looks like the `cmake` portion is 1-time setup. Once it's run successfully, the only thing you need to run tests is `make`. I think it's reasonable to treat the `cmake` portion as user-required setup/installation and the actual test run as `make`. I'll update accordingly.